app/views/register-your-interest/enter-your-email-address.njk

```diff
@@ -18,12 +18,13 @@
 
       {% call govukFieldset({
         legend: {
-          text: "Enter your email address",
+          text: "Enter your business email address",
           classes: "govuk-fieldset__legend--l",
           isPageHeading: true
         }
       }) %}
-        <p class="govuk-body">Enter the email address linked to your Rural Payments account. We'll use this to send you confirmation you’ve successfully registered your interest.</p>
+        <p class="govuk-body">Enter the business email address linked to the business registering an interest. This may be different to your personal email address which may also be linked to the business.</p>
+        <p class="govuk-body">We'll use this to send you confirmation you’ve successfully registered your interest.</p>
```

Comment:
In other place we used a different char ` instead of '

<img width="291" alt="image" src="https://user-images.githubusercontent.com/118533775/208386713-43aad430-aca1-4347-a68e-74dc68c3de7f.png">